src/main/java/com/example/onlinebookstore/dto/CategoryDto.java

```diff
@@ -0,0 +1,12 @@
+package com.example.onlinebookstore.dto;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+import jakarta.validation.constraints.NotBlank;
+
+@Schema(description = "Category response DTO")
+public record CategoryDto(
+        @NotBlank
```

Comment:
Do we need validation here if it's response?

Reply:
o.. really don't need!..thank you